applications/matter_weather_station/README.rst

```diff
@@ -203,6 +244,9 @@ After building the target, the generated :file:`factory_data.hex` file will be m
 If you want to use Vendor ID, Product ID or other data that is not reserved for tests, you need custom test certificates.
 To build with custom certificates, you need to :ref:`install CHIP Certificate Tool <ug_matter_gs_tools_cert_installation>`.
 
+.. note::
+   Currently, the factory data feature is not supported for the Wi-Fi variant of the Matter Weather Station application.
```

Comment:
Actually why is it not supported? I have never tested it, but I assume there should not be difference in partitions layout.